### PR TITLE
be-1907 Automatically set pm.max_children in php-fpm.

### PR DESCRIPTION
**Pull request by mbarcia**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -35,3 +35,71 @@
...
+      state: installed
+  with_items:
+    - python-dev
+    - python-setuptools
+    - python-virtualenv
+    - python-ldap
+    - libsasl2-dev
+    - libldap2-dev
+    - libssl-dev
+    - libffi-dev
```
Comment (codesplicer) [inline tasks/main.yml]: don't need python-ldap and the rest of the dependencies below

File: files/fpm_dimensioning.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+'''
+Set PHP-FPM pm.max_children in order to maximize usage of available memory in the box at startup time.
+
+'''
+import os
+import logging
+import time
```
Comment (codesplicer) [inline files/fpm_dimensioning.py]: Don't need the time import as it's not being used

File: files/fpm_dimensioning.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    return mem_megs * php_fpm_total_ram_slice
+
+@argh.arg("--slice", help="Slice of RAM allocated to PHP-FPM. 0 means no RAM, 1 means all the RAM available. Defaults to 0.4.", default=0.4)
+@argh.arg("--confpath", help="Pool configuration file path. Defaults to /etc/php5/fpm/pool.d/www.conf.", default="/etc/php5/fpm/pool.d/www.conf")
+@argh.arg("--avgmem", help="Estimated safe average memory per thread (in megs). Defaults to 128.", default=128)
+def write_max_children_config(**kwargs):
+    '''Overwrites the max children value in the default PHP-FPM config
+    '''
+
+    slice = kwargs['slice']
```
Comment (codesplicer) [inline files/fpm_dimensioning.py]: slice is a python keyword, this needs to be renamed

Comment (mbarcia) [inline files/fpm_dimensioning.py]: Done

File: README.md [hunk 4]
```diff
@@ -11,17 +12,27 @@ Installs the latest PHP5 and associated packages on an ubuntu host
...
+    - { role: webhop.php5, fpm_avgmem: 90 }
 ```
 
 Author Information
 -------------------
 
 * Chris Warren - chris@beamly.com
 * Neil Saunders - neil@beamly.com
+* Mariano Barcia - mariano.barcia@beamly.com
+* Vik Batthi - vik@beamly.com
```
Comment (codesplicer) [inline README.md]: Misspelling, its 'Vik Bhatti'

**Pull request merged**